src/sst/index.ts

```diff
@@ -78,7 +78,7 @@ export class OttofellerSSTProject extends TypeScriptAppProject {
     }
 
     // ANCHOR ESLint and prettier setup
-    const lintPaths = options.lintPaths ?? ['.projenrc.ts', 'src', 'sst.config.ts']
+    const lintPaths = options.lintPaths ?? ['.projenrc.ts', 'stacks', 'sst.config.ts']
```

Comment:
@quesabe do you think we can keep the `src/` as optional path? Sometimes TS will be stored there (rarely though).

Reply:
It might be added as an option. Something of that sort: `options.sourceCodeFolder: 'src' // defaults to 'stacks'`. However I'm not sure it's very useful, since one can just set his own `lintPaths` and have anything linted. I don't like different options that do the same thing, but if it's more convenient for those using the template, why not...

Let me know if we need such an option.

Reply:
Agree, let's stick to `lintPaths`.